packages/vitest/src/integrations/vi.ts

```diff
@@ -12,6 +12,7 @@ import { fn, isMockFunction, spies, spyOn } from './spy'
 import { waitFor } from './wait'
 
 interface VitestUtils {
+  isFakeTimers: () => boolean
```

Comment:
Let's keep the syntax similar to other methods:

```suggestion
  isFakeTimers(): boolean
```